airflow-core/src/airflow/ui/src/pages/Run/Run.tsx

```diff
@@ -32,12 +33,16 @@ export const Run = () => {
   const { t: translate } = useTranslation("dag");
   const { dagId = "", runId = "" } = useParams();
 
+  // Get external views with dag_run destination
+  const externalTabs = usePluginTabs("dag_run");
+
   const tabs = [
     { icon: <MdOutlineTask />, label: translate("tabs.taskInstances"), value: "" },
     { icon: <FiDatabase />, label: translate("tabs.assetEvents"), value: "asset_events" },
     { icon: <MdOutlineEventNote />, label: translate("tabs.auditLog"), value: "events" },
     { icon: <FiCode />, label: translate("tabs.code"), value: "code" },
     { icon: <MdDetails />, label: translate("tabs.details"), value: "details" },
+    ...externalTabs,
```

Comment:
Never thought about this before... but would we put plugins always to back or to front?